File: SourceCode/Src/Dorisoy.Pan.Data/Resources/UserResource.cs
namespace Dorisoy.Pan.Data.Resources
{
    public class UserResource : ResourceParameter
    {
        public UserResource() : base("Email")
        {
        }

        public string Name { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string IsActive { get; set; }
        public string  FolderId { get; set; }
        public string DocumentId { get; set; }
        public string Type { get; set; }

    }
}

File: UI/src/app/core/domain-classes/user-resource.ts
import { ResourceParameter } from './resource-parameter';

export class UserResource extends ResourceParameter {
  email: string = '';
  first_name: string = '';
  last_name: string = '';
  phone_number: string = '';
  is_active: boolean = true;
  folderId: string = '';
  documentId: string = '';
  physicalFolderId: string = '';
  type: string;
}
